tests/lib/dirs.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+
+# Default applies for: Ubuntu, Debian
+SNAPMOUNTDIR=/snap
+LIBEXECDIR=/usr/lib
+CORELIBEXECDIR=/usr/lib
```

Comment:
But in shell this introduces clutter and non-obviousness. It can also expand to nothing (AFAIR we don't run our tests with `-u` set) if there's a typo.